--- build.sh.bak	2026-04-20 16:31:07
+++ build.sh	2026-04-20 16:34:13
@@ -517,8 +517,11 @@
       if [[ ${FILE} != */MacOS/mkv* ]] non_executables+=(${FILE})
     }
 
+    harden=""
+    if [[ -n ${NOTARY_PROFILE} ]] harden="--options=runtime"
+
     codesign --force --sign ${SIGNATURE_IDENTITY} ${non_executables}
-    codesign --force --sign ${SIGNATURE_IDENTITY} ${dmgmac}/mkv*(.)
+    codesign --force ${harden} --sign ${SIGNATURE_IDENTITY} ${dmgmac}/mkv*(.)
   fi
 
   if [[ -n $DMG_NO_DMG ]] return
@@ -553,6 +556,10 @@
 
   if [[ -n ${SIGNATURE_IDENTITY} ]] codesign --force -s ${SIGNATURE_IDENTITY} ${dmgname}
 
+  if [[ -n ${NOTARY_PROFILE} ]]; then
+    xcrun notarytool submit ${dmgname} --keychain-profile ${NOTARY_PROFILE} --wait
+  fi
+
   if [[ ${dmgname} != ${dmgbuildname} ]] mv ${dmgname} ${dmgbuildname}
 
   ln -s ${dmgbuildname} ${latest_link}
