--- build.sh.bak	2026-04-18 14:44:46
+++ build.sh	2026-04-19 17:23:54
@@ -520,9 +520,15 @@
     for FILE (${dmgcnt}/**/*(.)) {
       if [[ ${FILE} != */MacOS/mkv* ]] non_executables+=(${FILE})
     }
+
+    if [[ -n ${NOTARY_PROFILE} ]] harden="--options runtime"
 
     codesign --force --sign ${SIGNATURE_IDENTITY} ${non_executables}
-    codesign --force --sign ${SIGNATURE_IDENTITY} ${dmgmac}/mkv*(.)
+    codesign --force ${harden} --sign ${SIGNATURE_IDENTITY} ${dmgmac}/mkv*(.)
+  fi
+
+  if [[ -n ${NOTARY_PROFILE} ]]; then
+    xcrun notarytool submit ${dmgapp} --keychain-profile ${NOTARY_PROFILE} --wait
   fi
 
   if [[ -n $DMG_NO_DMG ]] return
