--- specs.sh.bak	2026-04-21 10:05:05
+++ specs.sh	2026-04-21 09:54:23
@@ -12,6 +12,6 @@
 spec_ogg=(libogg-1.3.4.tar.xz http://downloads.xiph.org/releases/ogg/libogg-1.3.4.tar.xz c163bc12bc300c401b6aa35907ac682671ea376f13ae0969a220f7ddf71893fe)
 spec_pkgconfig=(pkg-config-0.29.2.tar.gz https://pkg-config.freedesktop.org/releases/pkg-config-0.29.2.tar.gz 6fc69c01688c9458a57eb9a1664c9aba372ccda420a02bf4429fe610e7e7d591)
 spec_vorbis=(libvorbis-1.3.7.tar.xz http://downloads.xiph.org/releases/vorbis/libvorbis-1.3.7.tar.xz b33cc4934322bcbf6efcbacf49e3ca01aadbea4114ec9589d1b1e9d20f72954b)
-spec_zlib=(zlib-v1.3.1.tar.xz https://zlib.net/zlib-1.3.1.tar.xz 38ef96b8dfe510d42707d9c781877914792541133e1870841463bfa73f883e32)
+spec_zlib=(zlib-v1.3.2.tar.xz https://zlib.net/zlib-1.3.2.tar.xz d7a0654783a4da529d1bb793b7ad9c3318020af77667bcae35f95d0e42a792f3)
 
-spec_qt=(qt-everywhere-src-6.10.0.tar.xz https://download.qt.io/official_releases/qt/6.10/6.10.0/single/qt-everywhere-src-6.10.0.tar.xz 81895fb038a9c3d6c6f698d7611339a189eb45c3d91746c7789b0b77a5981aa3)
+spec_qt=(qt-everywhere-src-6.11.0.tar.xz https://download.qt.io/archive/qt/6.11/6.11.0/single/qt-everywhere-src-6.11.0.tar.xz acf3b3db04c9e5d0820e8324b097320388954c297cee83d2bd698789234f68a4)
