--- config.sh.bak	2026-04-21 10:20:12
+++ config.sh	2026-04-21 09:54:23
@@ -14,6 +14,7 @@
 export MACOSX_DEPLOYMENT_TARGET="13"
 export DRAKETHREADS=${DRAKETHREADS:-4}
 export MAKEFLAGS="-j ${DRAKETHREADS}"
-export QTVER=${QTVER:-6.10.0}
+export QTVER=${QTVER:-6.11.0}
 export SIGNATURE_IDENTITY="Developer ID Application: Moritz Bunkus (YZ9DVS8D8C)"
 export DYLD_LIBRARY_PATH=${TARGET}/lib:${DYLD_LIBRARY_PATH}
+export NOTARY_PROFILE=""
